build.sbt

```diff
@@ -100,12 +90,11 @@ dockerCommands := dockerCommands.value.flatMap {
   case cmd @ Cmd("ADD", _) =>
     List(
       Cmd("RUN", "adduser -u 2004 -D docker"),
-      Cmd("ENV", s"TOOL_VERSION ${toolVersion.value}"),
-      Cmd("ENV", s"STYLELINT_CONFIG_BASEDIR /usr/lib/node_modules"),
+      Cmd("ENV", s"STYLELINT_CONFIG_BASEDIR $defaultDockerInstallationPath/node_modules"),
       cmd,
-      Cmd("RUN", installAll()),
-      Cmd("RUN", "mv /opt/docker/docs /docs"),
+      Cmd("RUN", s"mv $defaultDockerInstallationPath/docs /docs"),
       ExecCmd("RUN", Seq("chown", "-R", s"docker:docker", "/docs"): _*),
```

Comment:
```suggestion
      ExecCmd("RUN", "chown", "-R", "docker:docker", "/docs"),
```

Reply:
You can even use Cmd here too for consistency

Reply:
Done ✅